lib/shared/utils/route_definitions.dart

```diff
@@ -0,0 +1,7 @@
+import '../../movements/view/movements_page.dart';
+import '../../portifolio/view/portfolio_page.dart';
+
+var routes = {
+  "/portifolio": (context) => const PortfolioPage(),
+  "/movements": (context) => const MovementsPage(),
+};
```

Comment:
MaterialApp(
  // Provide a function to handle named routes.
  // Use this function to identify the named
  // route being pushed, and create the correct
  // Screen.
  onGenerateRoute: (settings) {
    // If you push the PassArguments route
    if (settings.name == PassArgumentsScreen.routeName) {
      // Cast the arguments to the correct
      // type: ScreenArguments.
      final args = settings.arguments as ScreenArguments;

      // Then, extract the required data from
      // the arguments and pass the data to the
      // correct screen.
      return MaterialPageRoute(
        builder: (context) {
          return PassArgumentsScreen(
            title: args.title,
            message: args.message,
          );
        },
      );
    }
    // The code only supports
    // PassArgumentsScreen.routeName right now.
    // Other values need to be implemented if we
    // add them. The assertion here will help remind
    // us of that higher up in the call stack, since
    // this assertion would otherwise fire somewhere
    // in the framework.
    assert(false, 'Need to implement ${settings.name}');
    return null;
  },
)

faz algo parecido com isso

Reply:
Eu adicionei a mudança de rotas no card 5. Posso fazer merge nessa branch?

Reply:
Assim eu não preciso refatorar dois cards